support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -431,30 +436,57 @@ function OneTimeCheckoutComponent({
 						recaptchaToken: recaptchaToken ?? '',
 						paymentMethodId: paymentMethodResult.paymentMethod.id,
 					};
-					const paymentResult = await processStripePaymentIntentRequest(
+					paymentResult = await processStripePaymentIntentRequest(
 						stripeData,
 						handle3DS,
 					);
-					if (paymentResult.paymentStatus === 'failure') {
-						setErrorMessage('Sorry, something went wrong.');
-						setErrorContext(appropriateErrorMessage(paymentResult.error ?? ''));
-					}
-					if (paymentResult.paymentStatus === 'success') {
-						const order = {
-							firstName: '',
-							paymentMethod: paymentMethod,
-						};
-						setThankYouOrder(order);
-						const thankYouUrlSearchParams = new URLSearchParams();
-						thankYouUrlSearchParams.set('contribution', finalAmount.toString());
-						window.location.href = `/${geoId}/thank-you?${thankYouUrlSearchParams.toString()}`;
-					}
 				}
-				setIsProcessingPayment(false);
+			}
+
+			setThankYouOrder({
+				firstName: '',
+				paymentMethod: paymentMethod,
+			});
+			const thankYouUrlSearchParams = new URLSearchParams();
+			thankYouUrlSearchParams.set('contribution', finalAmount.toString());
+			const thankYouHref = paymentResultThankyouRoute(
```

Comment:
This `thankYouHref` could be the paypal payment site right? Reads a bit confusingly in that sense, maybe a more generic name makes sense like "next step" or something

Reply:
Ye, was'nt particularly descriptive in it operation, habit of mine! Renamed to `nextStepRoute`